From cfef684e2c2c93c4a871d6c2a7af8f4c1b7c6741 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Tue, 18 Mar 2025 09:29:58 -0700
Subject: [PATCH] Fix the type for sighandler_t

__sighandler_t is private interface in glibc so its
better to not use it, as an aside fixes build with musl

Upstream-Status: Submitted [https://lists.gnu.org/archive/html/bug-time/2025-03/msg00000.html]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 src/time.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/time.c b/src/time.c
index 7ff29a4..82eb66a 100644
--- a/src/time.c
+++ b/src/time.c
@@ -728,7 +728,7 @@ run_command (cmd, resp)
      RESUSE *resp;
 {
   pid_t pid;			/* Pid of child.  */
-  __sighandler_t interrupt_signal, quit_signal;
+  sighandler_t interrupt_signal, quit_signal;
   int saved_errno;
 
   resuse_start (resp);
